site/gatsby-site/src/components/forms/SubmitForm.js

```diff
@@ -40,10 +47,13 @@ const SubmitForm = () => {
       if (incident.incident_id) {
         updateOne({ incident_id: incident.incident_id }, values);
       } else {
-        await user.functions.createReportForReview(values);
+        await user.functions.createReportForReview({
+          ...values,
+          incident_id: 0,
```

Comment:
I believe you have this reversed and the default setting for incident_id should be listed before the values object is unpacked.
```
a = {"1":1}
{1: 1}
{...a}
{1: 1}
{...a, "1":2}
{1: 2}
{...a, "1":2}
{1: 2}
```